packages/next/src/build/webpack/loaders/next-app-loader.ts

```diff
@@ -302,9 +304,18 @@ async function createTreeCodeFromPath(
 
       const parallelSegmentPath = subSegmentPath.join('/')
 
+      const fileTypesToResolve: (FileTypeValues | 'page')[] =
```

Comment:
since we're creating a separate type for this, I think it'd make more sense to add the `| 'page'` to `FileTypeValues` rather than inlining it